int8 leftWheel
int8 rightWheel